SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Utils/EntityFactory/EntityFactoryTest.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+
+#include "Utils/EntityFactory.h"
+
+#include "CoreMinimal.h"
+
+#define ENTITYFACTORY_TEST(TestName) GDK_TEST(Core, EntityFactory, TestName)
+
+using namespace SpatialGDK;
+
+ENTITYFACTORY_TEST(GIVEN_an_actor_WHEN_creating_skeleton_entity_components_THEN_it_contains_required_components)
```

Comment:
This is an example test, and we can either expand this one or add similar ones.
CreateSkeletonEntityComponents is a nicely unit-testable function, but it's also kind of useless to test, I feel.
Let me know if tests like this would provide value or if I should try completely different ones, maybe trying to test addLBcomps.